packages/workflow/src/Workflow.ts

```diff
@@ -248,11 +260,15 @@ export class Workflow {
 			key = 'global';
 		} else if (type === 'node') {
 			if (node === undefined) {
-				throw new Error(`The request data of context type "node" the node parameter has to be set!`);
+				throw new Error(
+					`The request data of context type "node" the node parameter has to be set!`,
```

Comment:
A rule to ban unneeded backticks would be nice.

Reply:
there is just a rule for consistency but not for disablement at all. I mean `'` `"` and also ``` are all valid string operators in JS. So this would need to be checked by the reviewer or accepted